app/cli/cmd/attestation_verify.go

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright 22025 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/chainloop-dev/chainloop/app/cli/internal/action"
+	"github.com/spf13/cobra"
+)
+
+func newAttestationVerifyCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "verify file-or-url",
+		Short:                 "verify an attestation",
+		Long:                  "Verify an attestation by validating its validation material against the configured trusted root",
+		DisableFlagsInUseLine: true,
+		Example: `  # verify local attestation
+  chainloop attestation verify attestation.json
+
+  # verify an attestation stored in an https endpoint
+  chainloop attestation verify https://myrepository/attestation.json`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			res, err := action.NewAttestationVerifyAction(actionOpts).Run(cmd.Context(), args[0])
```

Comment:
note that this is the first command that we do that relies on arguments and no flags, if we want to be consistent, we should use a flag i.e `-b` or `--bundle`. 

I am not saying that using an argument is wrong, in fact I have mixed feelings about our current approach, but I am just pointing out the inconsistency. What's your take?

In the future we might want to also allow passing the root so they do not need to contact the controlplane, but that can come later.

Reply:
Yes, make sense, I'll implement the change.